### PR TITLE
Lasgweloth

### PR DESCRIPTION
**Pull request by Gondolindrim**
Lasgweloth v1.0 support

## Description

Adds support for Lasgweloth v1.0 PCB

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/acheron/lasgweloth/config.h [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4150 // AP for AcheronProject
+#define PRODUCT_ID      0x7641 // LA for Lasgweloth
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    AcheronProect
+#define PRODUCT         Lasgweloth
+#define DESCRIPTION	A leaf-spring tray mount universal 60 percent
```
Comment (drashna) [inline keyboards/acheron/lasgweloth/config.h:26]:
```suggestion
```

File: keyboards/acheron/lasgweloth/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#define ____ KC_NO
```
Comment (drashna) [inline keyboards/acheron/lasgweloth/keymaps/default/keymap.c:17]: These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

File: keyboards/acheron/lasgweloth/keymaps/xoxotus/keymap.c [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#define ____ KC_NO
```
Comment (drashna) [inline keyboards/acheron/lasgweloth/keymaps/xoxotus/keymap.c:17]: These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/acheron/lasgweloth/rules.mk [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
```
Comment (fauxpark) [inline keyboards/acheron/lasgweloth/rules.mk:19]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/acheron/lasgweloth/keymaps/xoxotus/keymap.c [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+   [0] = LAYOUT(
+   	KC_GESC, KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC,
+        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_TILD, 
+        KC_LCTL, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT, KC_ENT , KC_DEL,
```
Comment (noroadsleft) [inline keyboards/acheron/lasgweloth/keymaps/xoxotus/keymap.c:25]: The Delete key at the end of this line, is it half of the split backspace? And if so, which half?

Comment (Gondolindrim) [reply keyboards/acheron/lasgweloth/keymaps/xoxotus/keymap.c:25]: Right half.

File: keyboards/acheron/lasgweloth/keymaps/xoxotus/keymap.c (hunk 3 above)
Comment (Gondolindrim) [reply keyboards/acheron/lasgweloth/keymaps/xoxotus/keymap.c]: Committed

File: keyboards/acheron/lasgweloth/keymaps/default/keymap.c (hunk 2 above)
Comment (Gondolindrim) [reply keyboards/acheron/lasgweloth/keymaps/default/keymap.c]: Committed

File: keyboards/acheron/lasgweloth/rules.mk (hunk 4 above)
Comment (Gondolindrim) [reply keyboards/acheron/lasgweloth/rules.mk:19]: Remove unnecessary comment lines on rules.mk

**Comment by Gondolindrim** [comment]
Changes applied, requesting re-review

File: keyboards/acheron/lasgweloth/readme.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+The Lasgweloth PCB is an Open-Hardware guidelines compliant PCB which files can be found at [this link](https://github.com/AcheronProject/Lasgweloth). Its designer and maintainer is [Gondolindrim](https://github.com/Gondolindrim).
+
+## Layouts
+
+The possible layouts consist of a default or 7U bottom row, with possibility of using split backspace, split right shift and stepped caps lock.
+
+## PCB Documentation
+
+See the [AcheronDocs](https://gondolindrim.github.io/AcheronDocs/lasgweloth/intro.html) page for the Lasgweloth full documentation. You can also check the KiCad PCB files at the [GitHub repository](https://github.com/AcheronProject/Lasgweloth).
```
Comment (drashna) [inline keyboards/acheron/lasgweloth/readme.md:19]: 404's

Comment (Gondolindrim) [reply keyboards/acheron/lasgweloth/readme.md]: The Acheron Project website was wrong link. The github links are fine, it's just that the repo is not public yet but will be once I ship the GB PCBs

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**